harness/types/capabilities.ts

```diff
@@ -48,6 +51,7 @@ let sdkCapabilities: { [key: string]: string[] } = {
         Capabilities.cloud,
```

Comment:
could you please change this to `local`

Reply:
wait what's wrong with java having cloud capabilities

Reply:
also there is no local option, seems to be the default

Reply:
oh wait sorry, not what i meant to comment on